docs/rpc@usage/rpcclient.py

```diff
@@ -0,0 +1,60 @@
+import Pyro4
+import sys
+
+class RPCClient(object):
+    def __init__(self):
+    #Loads the variable dict with the contents of config.json.
+        self.dict = {
+                   "function-list" : {
+                            "provision" : "4",
+                            "detach_node" : "1",
+                            "create_snapshot" : "2",
+                            "list_all_images" : "1",
+                            "list_snaps" : "1",
+                            "remove_snaps" : "2"
+                            }
+                     } 
+        self.func_list = self.dict['function-list'] #The script name and no. of arguments.
+    
```

Comment:
As discussed, can you just provide the comment for the future purpose of the dictionary within dictionary...